src/pretix/control/forms/item.py

```diff
@@ -172,6 +218,17 @@ def save(self, *args, **kwargs):
 
         return instance
 
+    def clean(self):
+        cleaned_data = super().clean()
+
+        if self.cleaned_data.get('quota_option') == '2':
+            if not self.cleaned_data.get('quota_add_new_name'):
+                raise forms.ValidationError(
+                    {'quota_add_new_name': ['Quota name is required.']}
```

Comment:
> I left the size not required, because that's how the Quota form functions now--you can leave size blank and it will just create a Quota of size unlimited. Maybe I should add help text for that?

:see_no_evil: I'm always surprised how bad I know my own system. You're obviously right.